Added sysv init script, updated README

jcostom opened the pull request:
The init script isn't the greatest ever, but it's getting the job done on my systems.  I'm running vanilla Ubuntu 14.04, which doesn't use systemd.

jcostom committed:
Added sysv init script, updated README

The init script isn't the greatest ever, but it's getting the job done on my systems.

cdhowie commented on netflix-no-ipv6-dns-proxy.init:
```diff
@@ -0,0 +1,75 @@
+    stop)
+	log_daemon_msg "Stopping Netflix domain name service fix..." "netflix-no-ipv6-dns-prox"
+
+	pid=$(ps ax|grep server.py |grep python | awk '{print $1}')
```
I'm a bit worried that this has the potential to kill other processes.  Perhaps you could use the `--make-pidfile /var/run/netflix-no-ipv6-dns-proxy.pid` option on line 30 to create a pid file, then instead of using ps/kill directly here, use `start-stop-daemon --stop --pidfile /var/run/netflix-no-ipv6-dns-proxy.pid --remove-pidfile`?

jcostom committed:
Fixed up hack-job init script. :)

Now properly uses pid files in /var/run to properly ID the process,
ensuring only the proxy is killed.

jcostom committed:
Fixed Up README
